scipy/stats/_continuous_distns.py

```diff
@@ -5800,6 +5800,12 @@ def _stats(self, nu):
         g2 /= nu*mu2**2.0
         return mu, mu2, g1, g2
 
+    def _fitstart(self, data, args=None):
+        # Empirically good first estimates
+        loc = data.mean()
```

Comment:
Please double-check, but I think `data` might not be a NumPy array at this point; e.g. it could be a list. If that's right, consider changing this to `np.mean(data)`

Reply:
Yes, we better use your proposition for safety. This is addressed in 7ba2e55ed61663cd5d6b1626c247439451e05253.